src/Controller/Admin/UserCrudController.php

```diff
@@ -42,6 +42,9 @@ public function configureFields(string $pageName): iterable
         yield TextField::new('name', 'Name')
             ->setFormTypeOptions(['disabled' => true])
         ;
+
+        yield TextField::new('initials', 'Initials');
+
```

Comment:
For consistency with the rest of the code you should remove the blank lines around this `yield` expression.

Reply:
Fixed in fdb167390884ab8ec0839c257fb7d2c343e83e6d.